recipes-domd/domd-image-weston/domd-image-weston.bbappend

```diff
@@ -75,7 +75,6 @@ python do_domd_install_machine_overrides() {
 
 XT_QUIRK_PATCH_SRC_URI_rcar = "\
     file://${S}/meta-renesas/meta-rcar-gen3/docs/sample/patch/patch-for-linaro-gcc/0001-rcar-gen3-add-readme-for-building-with-Linaro-Gcc.patch;patchdir=meta-renesas \
-    file://0001-hack-Remove-dependency-to-multi-user.target-from-wes.patch;patchdir=meta-renesas \
```

Comment:
I see you **remove** the patch, but I can't find where it is **moved** as the title says

Reply:
It was improperly rebased wis 62f05d1. Fixed.